docs/en/reference/working-with-associations.rst

```diff
@@ -500,8 +500,27 @@ and the "remove" operation.
         //...
     }
 
-Even though automatic cascading is convenient it should be used
-with care. Do not blindly apply cascade=all to all associations as
+Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
+association, you can now create a user and persist his comment as follows:
+
+.. code-block:: php
+
+    <?php
+    $user = new User();
+    $myFirstComment = new Comment();
+    $user->addComment($myFirstComment);
+    
+    $em->persist($user);
+    $em->flush();
+
+.. note::
+
+    This code does not associate the newly created comment with the user.
+    To achieve this, you *always* have to call ``$myFirstComment->setUser($user);`` –
+    no matter if ``cascade: persist`` is set or not.
```

Comment:
I changed several (hopefully all) occurances of "he" to gender neutral.

Broken example: Well, I wouldn't call the exmple broken. That's just how `cascade:persist` apparently works ;-)